src/improvements/tasks/sep/000-opcm-upgrade-v200/config.toml

```diff
@@ -0,0 +1,18 @@
+l2chains = [
+    {name = "OP Sepolia Testnet", chainId = 11155420}, 
+    {name = "Soneium Testnet Minato", chainId = 1946},
+    {name = "Ink Sepolia", chainId = 763373}
+]
+
+templateName = "OPCMUpgradeV200"
+
+[opcmUpgrades]
+absolutePrestates = [
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 11155420}, 
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 1946}, 
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 763373}
+]
+standardValidatorV200 = "0x37739a6b0a3F1E7429499a4eC4A0685439Daff5C"
+
+[addresses]
+OPCM = "0x1B25F566336F47BC5E0036D66E142237DcF4640b"
```

Comment:
I noticed that the schema in this file differs from others, [example](https://github.com/ethereum-optimism/superchain-ops/blob/823fa8b41782ea63d1a4dcc24be68abf292ab86a/src/improvements/tasks/eth/000-opcm-upgrade-v200/config.toml#L15).

Not sure how that affects things.

Reply:
Yea the other PRs still need to be updated, this is what it should look like.